tests/test_utils.py

```diff
@@ -99,7 +103,37 @@ def test_all_countries(self, unused_rglob_mock):
         we only run it once on the latest Python version.
         """
         warnings.simplefilter("ignore")
+
         for country in utils.list_supported_countries():
```

Comment:
I was just thinking, maybe it makes sense to test for all subdivisions (for countries that have them)? Or will it be excessive?
But anyway, LGTM.

Reply:
Tests help keep it compatible, stable and correct. I'm open to adding any non-redundant tests where necessary. It could be coupled with migration of code to "a method per a subdivision" approach if I understand you correctly.

We can discuss it to make sure everybody is on the same page, right?